ulfhara.lic

```diff
@@ -122,6 +123,23 @@ class UlfHara
         combo
       when /help any of our wounded you encounter/
         fput("accept task")
+        if DRStats.empath? && !DRSpells.active_spells.include?('Regenerate') && @settings.waggle_sets['healing']['Regenerate']
+          data = @settings.waggle_sets['healing']['Regenerate']
+          check_discern(data, @settings) if data['use_auto_mana']
+          release_cyclics
+          DRC.bput('release spell', 'You let your concentration lapse', "You aren't preparing a spell") unless checkprep == 'None'
+          DRC.bput('release mana', 'You release all', "You aren't harnessing any mana")
+          Flags.reset('spell-ready')
+          prepare?(data['abbrev'], data['mana'], data['symbiosis'], 'prep')
+        elsif DRStats.empath? && !DRSpells.active_spells.include?('Heal') && @settings.waggle_sets['healing']['Heal']
+          data = @settings.waggle_sets['healing']['Heal']
+          check_discern(data, @settings) if data['use_auto_mana']
```

Comment:
check_discern returns modified data

Reply:
yes, but it directly modifies it, not a local modification.

Reply:
`data = check_discern(data, @settings) if data['use_auto_mana']` makes more sense, since we'll be updating the data with the results from `check_discern`

Reply:
changed.

Reply:
nevermind! TIL that hashes are modified in place